client/src/index.jsx

```diff
@@ -20,7 +28,14 @@ class Header extends React.Component {
     let endpoint = '/funding/' + splitComponentUrl[3];
     axios.get(urlWithoutEndpoint + endpoint)
```

Comment:
I think it'll look a little cleaner if you just hard code in the base component server route since that will be the same no matter what e.g. `http://localhost:3004` but this works as well.